### DISCUSSION TITLE
Algorithms for graph drawing

### DISCUSSION THREAD

**Discussion by erelsgl** [networkx/networkx#5368] [General]
What algorithms for graph drawing are currently missing from `networkx`, and you think could be useful to add?

**Comment by MridulS** [comment]
One famous one which is missing is ForceAtlas2 (https://doi.org/10.1371/journal.pone.0098679). There was a recent attempt (https://github.com/networkx/networkx/pull/5311) to vendor an external library which implements ForceAtlas2 in Python but the code seems unmaintained and it's also GPL licensed.

Getting a pull request which implements ForceAtlas2 from scratch for NetworkX would be great!

There are some other discussions like https://github.com/networkx/networkx/discussions/4710 which also discuss adding new layouts for graph drawing.
